src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/LocalSlotMappingTests.cs

```diff
@@ -841,7 +841,7 @@ static async Task M()
         await using IAsyncDisposable <N:1>v = F()</N:1>, <N:0>u = F()</N:0>;
     }
 }");
-            var asyncStreamsTree = Parse(AsyncStreamsTypes);
+            var asyncStreamsTree = Parse(AsyncStreamsTypes, options: TestOptions.Regular.WithNoRefSafetyRulesAttribute());
```

Comment:
These two tests look very similar but in one we directly specify the options and in the other we copy the options from `source0`. Is there a reason for that?